pkg/scheduler/framework/plugins/examples/postbind/postbind.go

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Please remove this file. We should add example plugins only when there is important information that we want to show to plugin developers. I don't see any particularly important information in this example that is not already shown in existing example plugins.